src/containers/contract/index.js

```diff
@@ -132,9 +138,10 @@ class Contract extends PureComponent {
                   <div className="description Contract-content-item">{contract.data.description}</div>
                   {contract.data.status !== 4 && !contract.data.partyAFee && !contract.data.partyBFee ?
                     <div className="Contract-content-actions">
-                      <div className="Contract-content-actions-button Contract-actions-button-left" onClick={this.createDispute}>Create dispute</div>
-                      {contract.data.partyA === accounts.data[0] && <div className="Contract-content-actions-button Contract-content-actions-button-right" onClick={this.createPay}>Pay</div>}
-                      {contract.data.partyB === accounts.data[0] && <div className="Contract-content-actions-button Contract-content-actions-button-right" onClick={this.createReimburse}>Reimburse</div>}
+                      <div style={this.hideEmptyContractEl(contract)} className="Contract-content-actions-button Contract-actions-button-left" onClick={this.createDispute}>Create dispute</div>
+                      {contract.data.partyA === accounts.data[0] && <div style={this.hideEmptyContractEl(contract)} className="Contract-content-actions-button Contract-content-actions-button-right" onClick={this.createPay}>Pay</div>}
+                      {contract.data.partyB === accounts.data[0] && <div style={this.hideEmptyContractEl(contract)} className="Contract-content-actions-button Contract-content-actions-button-right" onClick={this.createReimburse}>Reimburse</div>}
+                      {this.showEmptyContractEl(contract) && <div className="Contract-content-item">No contractual fees remain</div>}
```

Comment:
the message is more "The contract has ended". 
Ideally it would be good to have the contract status as:
- Bob pays Alice
- Bob refunds Alice
- Bob wins the dispute
- Bob made a timeout